src/lib.rs

```diff
@@ -97,13 +97,20 @@ use serde::{de::DeserializeOwned, Serialize};
 use std::fmt;
 use std::sync::Arc;
 
-const PUBLIC_ENCRYPT_KEY_BYTES: usize = box_::PUBLICKEYBYTES;
-const SECRET_ENCRYPT_KEY_BYTES: usize = box_::SECRETKEYBYTES;
-const PUBLIC_SIGN_KEY_BYTES: usize = sign::PUBLICKEYBYTES;
-const SECRET_SIGN_KEY_BYTES: usize = sign::SECRETKEYBYTES;
-const SHARED_SECRET_KEY_BYTES: usize = box_::PRECOMPUTEDKEYBYTES;
-const SIGNATURE_BYTES: usize = sign::SIGNATUREBYTES;
-const SYMMETRIC_KEY_BYTES: usize = secretbox::KEYBYTES;
+/// Public encryption key length in bytes.
+pub const PUBLIC_ENCRYPT_KEY_BYTES: usize = box_::PUBLICKEYBYTES;
+/// Private encryption key length in bytes.
```

Comment:
I think we should stick to "secret" rather than "private" throughout (applies to line 106 too).